module/testmodule/main.go

```diff
@@ -142,13 +142,13 @@ func (h *helper) DoCommand(ctx context.Context, req map[string]interface{}) (map
 		msg := req["msg"].(string)
 		switch level {
 		case logging.DEBUG:
-			h.logger.Debug(msg)
+			h.logger.Debugw(msg, "foo", "bar")
```

Comment:
I understand this isn't a real module, but these can now be `CDebug(ctx, ...)` or `CInfo(ctx, ...)`

Reply:
Yep!